githubaction/0.1.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,27 @@
+# Artifact Hub package metadata file
+# https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 0.1.0
+name: githubaction-webhook
+displayName: GitHub Action Webhook Integration
+createdAt: 2021-11-17T00:00:00Z
+description: This integration shows how to invoke GitHub Action workflows leveraging Keptn's webhook service.
+logoURL: https://upload.wikimedia.org/wikipedia/commons/thumb/b/b9/Slack_Technologies_Logo.svg/2560px-Slack_Technologies_Logo.svg.png
+license: Apache-2.0
+homeURL: https://keptn.sh/docs/integrations/
+keywords:
+  - keptn
+  - webhook
+  - github
```

Comment:
please add `githubactions`

Reply:
fixed in https://github.com/keptn-sandbox/artifacthub/pull/34/commits/a0ab4afe44744d3ee98494d4f368c60a5f18bea4